submissions/5mountains/popup/css/main.css

```diff
@@ -0,0 +1,260 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    list-style-type: none;
+    text-decoration: none;
+    outline: none;
+    color: #000;
+}
+
+.nav {
+    margin: 1rem;
+}
+
+.nav__menu {
+    display: flex;
+    align-items: center;
+    justify-content: flex-end;
+}
+
+.nav__menu [type="checkbox"] {
+    display: none;
+}
+
+.menu__item,
+.menu__item-button {
+    padding-right: 1rem;
+}
+
+.menu__item:nth-child(1),
+.menu__item:nth-child(2) {
+    padding-right: .5rem;
+}
+
+.menu__item-input {
+    position: absolute;
+}
+
+.menu__item-input:checked ~ .menu__submenu,
+.menu__item-input:checked ~ .submenu__item-label {
+    display: flex;
+}
+
+.menu__item-input:checked + .menu__item-label::before {
+    opacity: 1;
+}
+
+.menu__item-label {
+    width: 30px;
+    height: 30px;
+    display: block;
+    cursor: pointer;
+    background-image: url("../img/menu/popup-button.png");
+    background-size: cover;
+}
+
+.menu__item-label::before {
+    opacity: 0;
+    content: "";
+    z-index: 10;
```

Comment:
Are you sure you need such a big `z-index`?

Could you try `z-index: 1` and tell me if it works ok? (:

Reply:
Of course not) because by default elements have 0/ So I just need put 1 extra unit to this value in order to change position (: But anyway, I will change it in proper value!

Reply:
My point was about such a big `z-index`
It's just for the future, some of your teammates could set `z-index: 999`, then you must set the biggest value for your needs and it will cause a lot of layout problems for you and your teammates
Let me know if it's clear)